Android.mk

```diff
@@ -56,6 +57,19 @@ ifeq ($(CFG_SECURE_DATA_PATH),y)
 srcs += sdp_basic.c
 endif
 
+define my-embed-file
+$(TARGET_OUT_HEADERS)/$(1).h: $(2)
+	@echo '  GEN     $$@'
+	@scripts/file_to_c.py --inf $$< --out $$@ --name $(1)
+
+$(call LOCAL_PATH)/host/xtest/regression_8100.c: $(TARGET_OUT_HEADERS)/$(1).h
+endef
+
+$(eval $(call my-embed-file,regression_8100_ca_crt,cert/ca.crt))
+$(eval $(call my-embed-file,regression_8100_mid_crt,cert/mid.crt))
+$(eval $(call my-embed-file,regression_8100_my_crt,cert/my.crt))
+$(eval $(call my-embed-file,regression_8100_my_csr,cert/my.csr))
```

Comment:
Seems like relative path `cert/*` doesn't work in aosp build env [1]. Can you try `$(LOCAL_PATH)/cert/*` please?

[1] https://ci.linaro.org/job/android-hikey-optee-o/204/consoleText